templates/member/admin/memberip.html

```diff
@@ -37,4 +37,25 @@
             {% endfor %}
         </ul>
     </div>
+
+    {# Checks if it's an IPV6 to show the members from the same IPV6 network #}
+    {% if ":" in ip %}Checks if it's an IPV6 to show the members from the same IPV6 network
```

Comment:
Une petite coquille s'est glissée là j'ai l'impression :)
```suggestion
    {# Checks if it's an IPV6 to show the members from the same IPV6 network #}
    {% if ":" in ip %}
```